### DISCUSSION TITLE
Account Hacked, Help Needed.

### DISCUSSION THREAD

**Discussion by abcuna** [community/community#116646] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I just received an email notification that my GitHub account has changed its password and email. I logged in with my passkey in an attempt to recover the email. Seconds later I was logged out with the passkey removed from my account, I tried contacting support but I am yet to get a response. I am willing to provide any source of evidence that the account is mine. Pls Help.

**Comment by AidenJamesYt** [comment]
really all you can do is open a support ticket, and try to access the account.

**Reply by abcuna** [reply]
I did that, but I’m yet to get a reply from them.

**Reply by AidenJamesYt** [reply]
It may take a few hours/days for your ticket to be reviewed. For now, all that you can really do is be patient.

**Comment by mateusloubach** [comment]
Don't you just find it weird that in a day and age we live in, with all the technology we have, literally ALL you can do is just open a ticket and pray that you'll get answered?? - This is absolutely nuts that we're still at this stage for countering hacks..

Here is the ONLY other possible way you could try to recover your account.. (also, the "best" option at this moment is contacting GitHub directly, as mentioned in the other answer). unfortunately, they don't leave us many options..

**IF** you had inserted a secondary email, you can try to change back your password. But make sure you don't insert any extra safekeys or 2FA at this point because you're trying to focus on revoking their access while granting yours. **keep in mind** you NEED to be fast on changing your acc password and focusing on revoking access to the other emails you don't know, as you do not know if they're currently accessing your acc and if alerted, could also revoke that other email.

I had lost access this week due to hack and was able to recover due to this.

**Reply by asmawi117** [reply]
same issue, mine got hacked 9 Feb i realize it coz some of my friends told me 3 days later, open my ticket at 12 Feb and still being ghosted by support team till now 😕

**Reply by mateusloubach** [reply]
I would like to update this due to the fact that my issue got taken care of.

When I first realized my github was hacked, I changed passwords, and added a secondary email to prevent me from being locked out. second thing I did was 2FA'd the heII out of it. (Authy, Google Authenticator, 1Passwords are all great options), and I set a pass key for whenever I need to authorize third parties. So I'm always informed of any changes, on my phone.

The next thing I did was open a Ticket for Github. - I've seen people post on the community that Github is taking MONTHS to answer, but I got my issue solved in two weeks. So, idk what the deal is. (my case was just my account got marked as SPAM, so maybe it was easier to deal with, hence the platform only blocks my accesses instead of banning my account, as it happened with my Reddit acc. (imo, this is the BEST approach).

Anyways, I'm still waiting for full acc recovery, since I was rejected from the Developer Program due to having been hacked and my Copilot revoked, but other than that, good as green.

Hope you get yours sorted out asap.

**Reply by abcuna** [reply]
I am still waiting for their response I even sent a message on LinkedIn to see if I can recover my acc or not, and yet, no response.
I about to give up.
